src/components/common/Button/Button.module.scss

```diff
@@ -29,12 +33,17 @@
 .outline {
   display: inline-block;
   padding: 5px 10px;
+  border: 1px solid #2a2a2a;
```

Comment:
Ogólnie w pliku settings mamy zmienne których powinniśmy używać do kolorów w plikach więc zamiast deklaracji kolorów używałbym zmiennych

Reply:
Zgadzam się, nie było to związane z tym taskiem ale faktycznie tak powinno być, poprawiłem wszystkie tego typu style i zastosowałem zmienne